Resources/doc/annotations/security.rst

```diff
@@ -54,6 +54,15 @@ Here is another example, making use of multiple functions in the expression::
     {
     }
 
+The message parameter allows you to customize the text shown to the user at the exception or error pages::
```

Comment:
I would just say: "The message parameter allows you to customize the exception message"

Exception messages are not meant to be displayed to the end user. I understand that it's easier to do it, but still not something I'd recommend.